cookbook/networking/fetch-data.md

```diff
@@ -0,0 +1,282 @@
+---
+layout: page
+title: "Fetch data from the internet"
+---
+
+Fetching data from the internet is necessary for most apps. Luckily, Dart and Flutter provide convenient tools to work with data from the internet!
+
+## Ingredients
+  
+  * [`http`](https://pub.dartlang.org/packages/http) package
+  * `convert` package (part of the Dart SDK)
+  * [`FutureBuilder`](https://docs.flutter.io/flutter/widgets/FutureBuilder-class.html) Widget
+  
+## Directions
+
+  1. Make a network request using the `http` package
+  2. Convert the response into a custom Dart object
+  3. Display the data
+  
+## 1. Make a network request
+
+The Dart Team publishes the [`http`](https://pub.dartlang.org/packages/http) package for making http requests. It provides the simplest way to fetch data from the internet.
```

Comment:
Is it important to state who publishes it?

(also, not sure if it's Dart Team. Maybe Dart team)

Reply:
It's Dart Team according to Pub

Reply:
Removed it. Let's keep the prose concise.